src/merkle-funder.ts

```diff
@@ -8,36 +8,50 @@ import { ChainConfig } from './types';
 export const fundChainRecipients = async (
   chainId: string,
   chainConfig: Pick<ChainConfig, 'options' | 'merkleFunderDepositories'>,
-  merkleFunderContract: ethers.Contract
+  merkleFunderContract: ethers.Contract,
+  logOptions: LogOptions
 ) => {
+  logger.info(`Processing ${chainConfig.merkleFunderDepositories.length} merkleFunderDepositories...`, logOptions);
+
   let nonce: number | null = null;
   for (const { owner, values } of chainConfig.merkleFunderDepositories) {
+    // TODO: should we compute the MerkleFunderDepository address and log it?
```

Comment:
👍 I've found it's useful having stuff like this (thinking of Airseeker sponsor wallets)